### ISSUE TITLE
I wish I could customize the background color

### ISSUE THREAD

**Issue by solowhale** [chetachiezikeuzor/Yin-and-Yang-Theme#62]
Hi, thank you for developing this theme, I really like the light version, but the background color of the dark version is too dark, it looks painful to the eyes. Can you add a feature to allow the Style Settings plugin to adjust the background color of the Yin-and-yang theme?

**Comment by chetachiezikeuzor** [comment]
Yes, this has already been added! If you have the most recent update installed, you should see a setting to change the background darkness value. Please let me know if you have any issues with this! ☺️

https://user-images.githubusercontent.com/79069364/136204214-ec3261c7-d654-491a-97ed-d7837697ebd4.mov

**Comment by solowhale** [comment]
> Yes, this has already been added! If you have the most recent update installed, you should see a setting to change the background darkness value. Please let me know if you have any issues with this! ☺️
> 
>  Screen.Recording.2021-10-06.at.7.41.10.AM.mov

ok,thank you.
